libraries/AP_Networking/AP_Networking.h

```diff
@@ -157,14 +157,23 @@ class AP_Networking
     enum class OPTION {
         PPP_ETHERNET_GATEWAY=(1U<<0),
 #if AP_NETWORKING_CAN_MCAST_ENABLED
-        CAN1_MCAST_GATEWAY=(1U<<1),
-        CAN2_MCAST_GATEWAY=(1U<<2),
+        CAN1_MCAST_ENDPOINT=(1U<<1),
+        CAN2_MCAST_ENDPOINT=(1U<<2),
+        CAN1_MCAST_BRIDGED=(1U<<3),
```

Comment:
These could also be protected by `AP_NETWORKING_CAN_MCAST_ENABLED` to catch places that are trying to handle bridged stuff when they shouldn't.

Reply:
> did you mean AP_NETWORKING_CAN_MCAST_BRIDGING_ENABLED ?

Yes, sorry